### PR TITLE
Windows module docs

### PR DESCRIPTION
**Pull request by trondhindenes**
##### ISSUE TYPE
- Docs Pull Request
##### COMPONENT NAME

Windows modules
##### ANSIBLE VERSION

```
2.2.0
```
##### SUMMARY

Added a definitive list of modules that will work with Windows. This wasn't documented in detail.

### CODE REVIEW

File: docsite/rst/intro_windows.rst [hunk 1]
```diff
@@ -246,7 +246,24 @@ What modules are available
 ``````````````````````````
 
 Most of the Ansible modules in core Ansible are written for a combination of Linux/Unix machines and arbitrary web services, though there are various
-Windows modules as listed in the `"windows" subcategory of the Ansible module index <http://docs.ansible.com/list_of_windows_modules.html>`_.
+Windows-only modules are listed in the `"windows" subcategory of the Ansible module index <http://docs.ansible.com/list_of_windows_modules.html>`_.
+
+In addition, the following core module work with Windows:
```
Comment (nitzmahone) [inline docsite/rst/intro_windows.rst]: s/module/modules

**Comment by nitzmahone** [comment]
one little fix, then LGTM

**Comment by trondhindenes** [comment]
Forgot about this one, fixed now. @nitzmahone, wold you hit merge if you're happy with the wording?

**Comment by dharmabumstead** [comment]
Thanks @trondhindenes!

**Pull request merged**